x-pack/plugins/security_solution/server/features.ts

```diff
@@ -116,16 +116,17 @@ export const getKibanaPrivilegesFeaturePrivileges = (ruleTypes: string[]): Kiban
   subFeatures: [],
   privileges: {
     all: {
-      app: [APP_ID, 'kibana'],
+      app: [APP_ID, 'csp', 'kibana'],
       catalogue: [APP_ID],
-      api: [APP_ID, 'lists-all', 'lists-read', 'lists-summary', 'rac'],
+      api: [APP_ID, 'lists-all', 'lists-read', 'lists-summary', 'rac', 'csp-all'],
       savedObject: {
         all: [
           'alert',
           'exception-list',
           'exception-list-agnostic',
           DATA_VIEW_SAVED_OBJECT_TYPE,
           ...savedObjectTypes,
+          'csp_rule',
```

Comment:
I suggest creating a constant in this file for the string `csp_rule` and mention in a comment that it needs to match the cloud security posture rule saved object type name